transcendence-app/src/game/dto/create-game.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { GameMode } from '../../../src/shared/enums/game-mode.enum';
+
+export class CreateGameDto {
+  @IsString()
+  @IsNotEmpty()
+  playerOneUsername!: string;
+
+  @IsString()
+  @IsNotEmpty()
+  playerTwoUsername!: string;
+
+  @IsNumber()
+  @IsNotEmpty()
+  gameDuration!: number;
+
+  @IsNumber()
```

Comment:
Unless I'm very mistaken, I think we could be more accurate here and use `@IsInt` validator rather than just `@IsNumber` here

Reply:
Ok, I will change it